src/mame/drivers/gticlub.cpp

```diff
@@ -1006,13 +1030,16 @@ void gticlub_state::gticlub(machine_config &config)
 	m_konppc->set_cbboard_type(konppc_device::CGBOARD_TYPE_GTICLUB);
 }
 
-void gticlub_state::thunderh(machine_config &config) //todo: add 68000 and K056230 from the I/O board
+void gticlub_state::thunderh(machine_config &config) // Todo: K056230 from the I/O board
```

Comment:
This is very bad place to put a TODO (which is all uppercases btw). Given the context, should be in the struct, preferably below the 68k declaration.